src/components/AppletsTable/AppletsTable.tsx

```diff
@@ -1,20 +1,48 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { t } from 'i18next';
 
+import { account } from 'redux/modules';
 import { Search } from 'components/Search';
 import { Table } from 'components/Table';
-import { headCells, rowsCells } from 'components/AppletsTable/mock';
+import { useTimeAgo } from 'hooks';
+import { Row } from 'components/Table/Table.types';
 
+import { headCells } from './AppletsTable.const';
 import { AppletsTableHeader, StyledButtons } from './AppletsTable.styles';
 
 export const AppletsTable = (): JSX.Element => {
-  const [rows, setRows] = useState(rowsCells);
+  const timeAgo = useTimeAgo();
+  const accData = account.useData();
+  const [rows, setRows] = useState<Row[] | undefined>(undefined);
+
+  useEffect(() => {
+    const formattedApplets = accData?.account?.applets?.map(({ displayName, updated }) => {
+      const lastEdited = timeAgo.format(new Date(updated));
+
+      return {
+        appletName: {
+          content: () => displayName,
+          value: displayName,
+        },
+        lastEdited: {
+          content: () => lastEdited,
+          value: lastEdited,
+        },
+        actions: {
+          content: () => '',
+          value: '',
+        },
+      };
+    });
+
+    formattedApplets && setRows(formattedApplets);
+  }, [accData?.account?.applets]);
 
   const handleSearch = (value: string) => {
-    const filteredRows = [...rowsCells].filter((row) =>
+    const filteredRows = rows?.filter((row) =>
```

Comment:
In the case of search, probably we should search not by "rows", but by some kind of global array with applets, which contains all elements, and not just those filtered.

Reply:
fixed